nixos/modules/security/pam.nix

```diff
@@ -777,11 +777,8 @@ in
           '';
 
         # Most of these should be moved to specific modules.
-        cups = {};
-        ftp = {};
         i3lock = {};
         i3lock-color = {};
-        screen = {};
         vlock = {};
         xlock = {};
         xscreensaver = {};
```

Comment:
The remaining ones would need custom modules -> That's why I leave them out for this pr.